### DISCUSSION TITLE
Requests to unpkg ?

### DISCUSSION THREAD

**Discussion by p1r4t3-s4il0r** [minio/minio#20615] [Q&A]
Hello,
I'm using minio since a few month and I'm really happy with it !

Last week i was checking my browser requests and I discovered that on the login page, the client on the browser is making calls to : `https://unpkg.com/detect-gpu@5.0.38/dist/benchmark/d-amd.json.`

I'm don't understand why. And is there any way to stop this ?

Thanks

**Comment by klauspost** [comment]
I believe the [console](https://github.com/minio/console) uses this to detect if the login animation can play.

You can try setting `CONSOLE_ANIMATED_LOGIN=off` env var, which may disable the check. If not, open an [issue](https://github.com/minio/console/issues).

**Reply by p1r4t3-s4il0r** [reply]
Thanks for your answer, I'll try it in the week and let you know.

**Reply by p1r4t3-s4il0r** [reply]
Well, it seems like it didn't work.
Is there an issue template ?

**Reply by klauspost** [reply]
Just click New Issue on the link above..
